pkg/crc/machine/start.go

```diff
@@ -44,7 +44,7 @@ const minimumMemoryForMonitoring = 14336
 func getCrcBundleInfo(bundlePath string) (*bundle.CrcBundleInfo, error) {
 	bundleName := filepath.Base(bundlePath)
 	bundleInfo, err := bundle.Use(bundleName)
-	if err == nil {
+	if err == nil && !bundle.IsCustomBundle(bundleName) {
```

Comment:
If this bit of code is there to always unpack custom bundles, I'd prefer to see it in a separate commit which explains why we are doing this.

Reply:
> > > These can be addressed in follow-ups to this PR, but I think at least this issue with ~/.crc/cache/crc_libvirt_4.7.5_custom/ never being overwritten needs to be fixed before bundle generation is released to users.
> > 
> > 
> > That means even nothing change on the `custom` bundle side we have to extract it freshly for each crc start?
> 
> Extracting custom bundle each time the user runs `crc start` is one easy way of dealing with it, but it's suboptimal :-/
> A better way of dealing with this would be to read the metadata file from the archive, and if this metadata does not match what is unpacked, then we unpack it again. But that's more work, not sure it fits in this sprint :)

Yes same what I discussed with @gbraad , I don't want to spend a lot of time on this if we are not getting enough user feedback if they are using bundle generation at all.

> Maybe there are other alternative ways, looking at file creation times? If the `.crcbundle` file was created after the cached directory, then recreate the directory?

It may fail on following scenario if user is not generating the bundle but consuming a generated bundle and downloaded/fetched without preserving existing timestamps. So I would go with extraction every time until we have a proper solution to detect if bundle dir need to be overwritten.

Reply:
> If this bit of code is there to always unpack custom bundles, I'd prefer to see it in a separate commit which explains why we are doing this.

@cfergeau Done.